textdescriptives/tests/test_extractor.py

```diff
@@ -43,3 +43,19 @@ def test_extract_df_error(nlp):
         td.extract_df(doc, metrics="not a metric")
     with pytest.raises(Exception) as e_info:
         td.extract_df(doc, metrics=True)
+
```

Comment:
It looks like test_extract_df_subsetters does not check these properly (i.e. they have no effect):
df[td.readability_cols]
    df[td.dependency_cols]
    df[td.descriptive_stats_cols]
    
apparently you can't correct old stuff.

Reply:
I'll check it out - thanks